lib/subscription-connection.js

```diff
@@ -83,6 +83,17 @@ module.exports = class SubscriptionConnection {
   }
 
   async handleConnectionInit (data) {
+    if (this.context instanceof Promise) {
+      try {
+        this.context = await this.context
+      } catch (err) {
+        this.fastify.log.error(err)
```

Comment:
These couple of lines are not being tested, but I'm not sure if it's completely safe to just simply remove the try/catch, just because of the catch in `lib/subscription.js`

Reply:
Nevermind, I just sent the message 'connection_init' in the intended test and it covered it. 100% coverage.